prusti-tests/tests/parse/ui/simple_forall.stderr

```diff
@@ -0,0 +1,14 @@
+error: expected one of: `for`, parentheses, `fn`, `unsafe`, `extern`, identifier, `::`, `<`, square brackets, `*`, `&`, `!`, `impl`, `_`, lifetime
```

Comment:
(For a different PR, but) maybe this error should be more specific? After the colon there, we should not be able to see most of what it lists.

Reply:
Could you open an "enhancement" issue for that? I'm not sure how hard it would be to implement.